filesystems/xfs/xfstests/README.md

```diff
@@ -0,0 +1,28 @@
+xfstests test suite
+=====================================
+Wrapper for xfstests test suite, see http://git.kernel.org/?p=fs/xfs/xfstests-dev.git;a=summary \
+Test Maintainer: [Xiong Murphy Zhou](mailto:xzhou@redhat.com)
+
+How to run it
+-------------
+Make sure you have at least 25G free disk space in this partition
+```bash
+$ df -H .
+```
+
+Clone the tests-beaker repository
+```bash
+$ git clone https://github.com/CKI-project/tests-beaker.git
+```
+
+Configure which test cases to run, e.g.
+```bash
+$ export TEST_PARAM_FSTYPE=xfs
+$ export TEST_PARAM_RUNTESTS=generic/001
+```
+
+Execute the test
+```bash
+$ cd filesystems/xfs/xfstests/
```

Comment:
Similarly, it's not a good idea to assume the global test location in a local README.md. Just drop this line :)

Reply:
Removed